source/changelog.txt

```diff
@@ -23,7 +23,7 @@ v1.5.0
 
 - ``mongosh`` now uses Node.js version 16.x.
 
-- ``mongosh`` no longer distributes per-distro``mongosh` linux packages. You
+- ``mongosh`` no longer distributes per-distro ``mongosh`` linux packages. You
```

Comment:
"distro" is short-hand / not Global English. Suggest rewording to the following:

```suggestion
- ``mongosh`` no longer provides per-distribution ``mongosh`` linux packages. You
```